### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by hb020**
Readme and Sphinx documentation cleaned up.
Was tested via local generation with "make html".
Should be ready for publication on spicelib.readthedocs.io, but I have no idea how that works.

### CODE REVIEW

File: doc/utilities/asc_to_qsch.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
-asc_to_qsch.exe
+asc_to_qsch
```
Comment (nunobrum) [inline doc/utilities/asc_to_qsch.rst:1]: This is really a .exe file that is installed by the Pip installer when the PyPi version is used.

File: doc/utilities/histogram.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-histogram.exe
+histogram
```
Comment (nunobrum) [inline doc/utilities/histogram.rst:1]: also a .exe. See previous comment

**Review by nunobrum — APPROVED** [review]
Great job fixing the documentation. The only thing that I will have to reedit later, is the command line tools that are created by the pip installer. They are really a .exe. 

Anyway, I think it is easier to just accept this pull-request, and make a quick edit afterwards.

**Comment by hb020** [comment]
yes, they are .exe, except on linux or macos, where there is no extension. So it should read "(.exe)". The explanation of that is already in the readme.

**Comment by hb020** [comment]
it looks like none of the proposed changes got merged.

**Comment by hb020** [comment]
This should be good.
the exe stuff is in there as well.

**Pull request merged**